contents/blog/how-to-start-a-growth-team.md

```diff
@@ -0,0 +1,123 @@
+
+## Define your team values
+
+In Thomas's opinion, one of the most effective things they did was to create a set of core team values. They were the team's compass: guiding decisions, settling debates, and driving overall culture.
+
+Thomas shares their values:
+
+### Data over debate
+
+By focusing on hard numbers and concrete evidence, you can set aside biases and focus on effective solutions.
+
+If data was not available, Thomas and his team would often ship a small experiment to gather data and keep the team moving forward. They realized that building and learning from a bare-bones MVP is much quicker than having frustrating debates over what to build.
+
+### Everything should be an experiment
+
+We already mentioned the benefits of experimenting everything, and Thomas and his team anchored this into their values. Thomas stresses that this did not mean everything needed to be an A/B test, but rather that every feature or change should have a plan:
+
+- What is the hypothesis of the change?
+- Who is the change targeting?
+- What metrics you will you look at?
+- How is the experiment going to be set up?
+- How long will it run for?
+- What do you plan to learn from it?
+- What are the potential risks or downsides?
+
+This way, every decision was backed by a thoughtful strategy – not just a shot in the dark. 
+
+### Deep empathy for customers
+
+It's possible to achieve your goal yet harm your user experience. The most famous example of this is from **Booking.com**. They increased sales by showing *anxiety-inducing* messages – such as "*32 customers are looking at the same hotel as you*" and "*another person is considering booking a room at this accommodation*". Thankfully, they stopped after they were [fined $7.5M](https://www.gvh.hu/en/press_room/press_releases/press-releases-2020/gigantic-fine-imposed-on-booking.com-by-the-gvh)!
+
+Let's Do This avoid such pitfalls by putting their users at the heart of how they build – even if it means sacrificing short-term gains.
+
+## Thomas's parting advice to growth engineers: Make your code changes as small as possible
+
+Growth engineering is all about learning quickly and iterating based on your findings. It's this mindset that Thomas feels is the most important for growth engineers to have. 
+
+He often repeated the following advice to his engineers:
+
+> "**80% of your tests will fail, so you need to get comfortable knowing your code will be deleted**. 
+> 
+> Applying this in practice means reducing the scope of your changes *while still* maximizing insights:
+>
+> For example, an engineer may scope a project and say it will take 2 weeks. It's important to challenge that and frame the question differently: Instead of asking 'How long will it take?', ask 'What can you do in only 1 week? 3 days? Half a day?". Often you'll find that you can reduce the scope of the change while still being able to validate your hypothesis".
```

Comment:
```suggestion
Growth engineering is all about learning quickly and iterating based on your findings. 

In Thomas' view, good growth engineers "need to get comfortable knowing their code will be deleted" because "80% of your tests will fail."

Applying this in practice means reducing the scope of your changes *while still* maximizing insights:

"An engineer may scope a project and say it will take 2 weeks. It's important to challenge that and frame the question differently.

"Instead of asking 'How long will it take?', ask 'What can you do in only 1 week? 3 days? Half a day?'. Often you'll find that you can reduce the scope of the change while still being able to validate your hypothesis".
```